app/src/main/java/com/example/studentarena/BaseActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.example.studentarena;
+
+import android.util.Log;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+//public class BaseActivity extends AppCompatActivity {
```

Comment:
do you need this file or was this added accidentally?

Reply:
Yes it was added accidentally, I wanted to debug for users permission